### ISSUE TITLE
message.get_args() doesn't work with custom commands_prefix

### ISSUE THREAD

**Issue by JdPaz** [aiogram/aiogram#574]
If you use a **custom commands_prefix** on the message_handler the method **message.get_args()** returns empty

Code sample:
```python
@dp.message_handler(commands=["image"], commands_prefix='*#')
async def sendImageCommand(message):
   args = message.get_args() 
   print(args) #empty args

@dp.message_handler(commands=["image"])
async def sendImageTwoCommand(message):
   args = message.get_args() 
   print(args) #everything is fine.
```

Invoke TelegramBot like:
`*image imagepath.jpg`

**Comment by JdPaz** [comment]
Solved in 2.13 version
```python
@dp.message_handler(commands=["image"], commands_prefix='*#')
async def sendImageCommand(message, command):
     print(command.args)
```

**Comment by evgfilim1** [comment]
Duplicates #269
